ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -187,6 +179,16 @@ const BaseSearchLinks = React.memo(({ variant, mainTranscript, genesById }) => {
     )
   }
 
+  if (geneNames && geneNames.length) {
+    let pubmedSearch = `(${geneNames.join(' OR ')})`
+    if (variations.length) {
+      pubmedSearch = `${pubmedSearch} AND ( ${variations.join(' OR ')})`
+      links.push({ name: 'google', href: `https://www.google.com/search?q=(${geneNames.join('|')})+(${variations.join('|')}` })
+    }
+
+    links.push({ name: 'pubmed', href: `https://www.ncbi.nlm.nih.gov/pubmed?term=${pubmedSearch}` })
```

Comment:
Is it possible to define a constant for the display order of the links? Moving the code to change the order is inconvenient for future changes.

Reply:
yeah I like that idea